seedlet/functions.php

```diff
@@ -269,6 +269,16 @@ function seedlet_setup() {
 				'enable_theme_default' => true,
 			)
 		);
+
+		// Add support for block templates (default template is content-only)
+		add_theme_support( 'block-templates' );
+		add_filter(
+			'block_editor_settings_all',
+			function( $settings ) {
+				$settings['defaultBlockTemplate'] = '<!-- wp:post-content /-->';
```

Comment:
As discussed on Slack p1628868491002800/1628867157.498300-slack-C029FM1EH, it may help to match alignments between template editor and view to wrap this in a group block:

```suggestion
				$settings['defaultBlockTemplate'] = '<!-- wp:group -->
				<div class="wp-block-group"><!-- wp:post-content /--></div>
				<!-- /wp:group -->';
```